### PR TITLE
Week 3 - Therese

### PR DESCRIPTION
**Pull request by ThereseHag**
I have done my best to create a business site, and to use my learnings from previous week. I decided to build with a mobile first approach, which was a lot easier this week. 

I decided to use a shadow box effect to make the site a bit more interesting, but wanted to keep it simple but elegant or something like that. I added some hover effects on links and some border radius to some of the boxes. I especially found that useful for the mobile version. 

I struggled a bit with getting the Javascript for the accordion to work, but it turned out to be easy to fix, as soon as I realized I should use ' ' instead of " " which I'm quite used to from HTML and CSS. I also had some problems getting the post to httpbin to work, but that also was an easy one, as soon as I realized the type for the submit button wasn't totally correct.

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,215 @@
...
+  .header {
+    background-image: url("header_medium.jpg");
+    border-radius: 0px;
+  }
+
+  .navigation-bar {
+    margin-top: 130px;
+  }
+
+  .placeholder {
```
Comment (HIPPIEKICK) [inline code/style.css:159]: Consider a column layout or allocating some more space as it's looking a little squished in a small tablet:

Skärmavbild 2021-02-05 kl  15 01 23

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,215 @@
...
+.accordion-content {
+  display: none;
+  color: black;
+  cursor: pointer;
+  margin: 10px;
+  border-top: 2px solid lightgray;
+  padding: 18px;
+  text-align: left;
+  outline: none;
+  transition: 0.4s;
```
Comment (HIPPIEKICK) [inline code/style.css:126]: Transitioning won't work with the display. [Further reference](https://stackoverflow.com/questions/3331353/transitions-on-the-css-display-property).

Comment (HIPPIEKICK) [reply code/style.css:126]: Also consider adding a `max-width` to avoid "jumping"

File: code/index.html [hunk 3]
```diff
@@ -3,28 +3,97 @@
...
+        </div>
       </div>
+    </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+    <footer class="footer">
+      <div class="footer-text">
+        <p>© Therese Hagelin
+          <br>
+          <br>
```
Comment (HIPPIEKICK) [inline code/index.html:87]: We can solve this with CSS instead! Divide your text into three `p` and give them margin :)

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi Therese!
Take a look at my comments on your form and also the responsiveness (your header content being too wide) in order to be approved.

Here you can also check out the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475).

File: code/index.html [hunk 4]
```diff
@@ -3,28 +3,97 @@
...
+        <a class="active" href="#Home">Home</a>
+        <a href="#Content">Content Writing</a>
+        <a href="#SoMe">Social Media</a>
+        <a href="#Customer">Customer Service</a>
+      </div>
+    </header>
+    <div class="placeholder">
 
-    <!-- Signup form -->
+    <div class="form-container">
```
[the remark this answers is not in the document]
Comment (ThereseHag) [reply code/index.html]: This has been removed, didn't need it :-)

File: code/index.html [hunk 5]
```diff
@@ -3,28 +3,97 @@
...
+        <label>
+          Other <input type="checkbox" value="otherservices" name="servicechoice"></label></input>
+        </label>
+        <p>Would you like help on a monthly basis or one time help?</p>
+        <label>
+          Monthly basis <input type="radio" value="monthly" name="packagetype" required></label></input>
+        </label>
+        <label>
+          One time <input type="radio" value="onetime" name="packagetype" required></label></input>
+        </label>
```
[the remark this answers is not in the document]
Comment (ThereseHag) [reply code/index.html]: This has been fixed.

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,215 @@
...
+  padding: 18px;
+  border: 4px solid lightgray;
+  border-radius: 20%;
+  display: inline-flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  height: 500px;
+  width: 100%;
+}
```
[the remark this answers is not in the document]
Comment (ThereseHag) [reply code/style.css]: I removed the margin for this. Thanks!

File: code/script.js [hunk 7]
```diff
@@ -1,9 +1,27 @@
...
-}
+const accordionHeaderOne = document.getElementById("accordion-header-1");
+const accordionContentOne = document.getElementById("accordion-content-1");
+const accordionHeaderTwo = document.getElementById("accordion-header-2");
+const accordionContentTwo = document.getElementById("accordion-content-2");
+const accordionHeaderThree = document.getElementById("accordion-header-3");
+const accordionContentThree = document.getElementById("accordion-content-3");
+console.log(accordionHeaderOne);
+console.log(accordionHeaderTwo);
+console.log(accordionHeaderThree);
```
[the remark this answers is not in the document]
Comment (ThereseHag) [reply code/script.js]: They are now removed. Thanks for reminding me :)

File: code/index.html [hunk 8]
```diff
@@ -3,28 +3,97 @@
...
+      </div>
+    </header>
+    <div class="placeholder">
 
-    <!-- Signup form -->
+    <div class="form-container">
+      <form class="form" action="https://httpbin.org/anything" method="POST">
+        <h3>Request your virtual assistance</h3>
+        <label>
+          Name:<input name="form-name" type="text" required></input>
```
[the remark this answers is not in the document]
Comment (ThereseHag) [reply code/index.html:25]: Fixed.

File: code/style.css (hunk 1 above)
Comment (ThereseHag) [reply code/style.css:163]: You are totally right, changed this to column.

File: code/index.html (hunk 3 above)
Comment (ThereseHag) [reply code/index.html]: Lovely, fixed now. Thanks a lot for a great advice.

File: code/style.css (hunk 2 above)
Comment (ThereseHag) [reply code/style.css]: Removed it.

**Comment by ThereseHag** [comment]
I have now tried to fix all. Thanks a lot for useful advice :)

File: code/style.css [hunk 9]
```diff
@@ -1,7 +1,219 @@
...
+
+.placeholder {
+  margin: 10px;
+  padding: 18px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 1400px;
```
Comment (HIPPIEKICK) [inline code/style.css:58]: Why so high? 🤔

File: code/index.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply code/index.html:25]: Just this one left then ☝️

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**